microprofile/galleon-common/src/main/resources/modules/system/layers/base/org/wildfly/extension/microprofile/telemetry-api/main/module.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2022 Red Hat, Inc. and/or its affiliates
+  ~ and other contributors as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.extension.microprofile.telemetry-api">
+
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${org.wildfly:wildfly-microprofile-telemetry-api}"/>
+    </resources>
+
+    <dependencies>
+        <module name="org.eclipse.microprofile.config.api"/>
+        <module name="org.wildfly.extension.microprofile.config-smallrye" />
+        <module name="org.wildfly.extension.opentelemetry"/>
+        <module name="org.wildfly.extension.opentelemetry-api" export="true"/>
+
+        <module name="io.opentelemetry.api" />
+        <module name="io.opentelemetry.context" />
+        <module name="io.smallrye.opentelemetry" export="true"/>
+
+        <module name="org.jboss.weld.api"/>
+        <module name="org.jboss.weld.core"/>
+        <module name="org.jboss.weld.spi"/>
+        <module name="org.wildfly.security.manager"/>
```

Comment:
Is the ```org.wildfly.security.manager``` module still needed - I have seen the usage has been reduced/removed? https://github.com/wildfly/wildfly/pull/16330/files#diff-fa31bee747f3994d63479897521a17e840769503bff3b09f86e83c5a57287773L16

Reply:
Updated